src/helpers.c

```diff
@@ -1199,3 +1200,17 @@ char *get_tracking_dir(void)
 {
 	return sys_path_join(globals.state_dir, "bundles");
 }
+
+bool confirm_action(const char *warning_msg, const char *action)
+{
+	int response;
+
+	if (warning_msg) {
```

Comment:
This doesn't need to be here. It's more flexible to remove this argument and the caller can print anything before calling confirm_action()

Reply:
done